### DISCUSSION TITLE
Concept of a "test stream" to set up new DJs – How to?

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6132] [Q&A] [answered]
Most commercial stream providers offer a "test stream" (same station, different input/output mounts, usually limited to 5 listeners or so). In manual setups using LiquidSoap/Icecast, this is also easily done.

This is typically used to help set up new DJs, without interrupting the station’s main stream(s).

If I understand AzuraCast’s concept correctly, I would have to set up a complete separate _station_ for this, right? Including much unneeded setup.

Or did I overlook something and we could have that _within the same station_ already?

**Comment by BusterNeece** [comment, marked as answer]
There is a "Clone Station" functionality in Station Management, and along with that, you can also instruct the system to clone the station and share the same storage location, so they both reference the same files (and those files are only processed once while being available on both stations).

**Comment by Moonbase59** [comment]
Ok, didn’t realize that. Cheers!
